tidb-lightning/tidb-lightning-backends.md

```diff
@@ -0,0 +1,465 @@
+---
+title: TiDB Lightning Import Modes
+summary: Learn the different import modes of TiDB Lightning.
+aliases: ['/docs/dev/tidb-lightning/tidb-lightning-tidb-backend/','/docs/dev/reference/tools/tidb-lightning/tidb-backend/','/tidb/dev/tidb-lightning-tidb-backend']
+---
+
+# TiDB Lightning Import Modes
+
+TiDB Lightning supports three backends: Importer-backend (default), Local-backend, and TiDB-backend. It determines how `tidb-lightning` delivers data into the target cluster.
+
+The **Importer-backend** (default) requires `tidb-lightning` to first encode the SQL or CSV data into KV pairs, and relies on the external `tikv-importer` program to sort these KV pairs and ingest directly into the TiKV nodes.
+
+The **Local-backend**: `tidb-lightning` first encodes data into key-value pairs, sorts and stores them in a local temporary directory, and writes these key-value pairs to each TiKV node in batches. Then, TiKV ingests these key-value pairs into the cluster. The implementation of Local-backend is the same with that of Importer-backend but does not rely on the external `tikv-importer` component.
+
+The **TiDB-backend** requires `tidb-lightning` to encode these data into SQL `INSERT` statements, and has these statements executed directly on the TiDB node.
```

Comment:
Seems not consistent with the original latest version: https://docs.pingcap.com/zh/tidb/dev/tidb-lightning-backends?

Reply:
I put "Importer-backend" before "Local-backend" because it is still the default mode. @glorv What do you think?